bitbybit/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## unreleased
+
+### Added
+
+- Added `defmt` support by providing two bitfield macro arguments: `defmt_bitfields` and
```

Comment:
this should be moved into 1.4.0 as that's the unreleased version (see https://crates.io/crates/bitbybit)

I'd like to merge once this patch has landed